ui/client/components/graph/node-modal/subprocess-input-definition/DndItems.tsx

```diff
@@ -0,0 +1,67 @@
+import {css} from "emotion"
+import update from "immutability-helper"
+import {cloneDeep} from "lodash"
+import React, {useCallback, useRef} from "react"
+import {Draggable, DraggableChildrenFn} from "react-beautiful-dnd"
+import {DragHandlerContext} from "../../../toolbarComponents/DragHandle"
+import {DragHandle} from "./DragHandle"
+import {DropTarget} from "./DropTarget"
+import {FakeFormWindow} from "./FakeFormWindow"
+import {ItemsProps} from "./Items"
+
+interface DndListProps<I> extends ItemsProps<I> {
+  disabled?: boolean,
+  onChange: (value: I[]) => void,
+}
+
+export function DndItems<I>(props: DndListProps<I>): JSX.Element {
+  const {items, onChange, disabled} = props
+
+  const moveItem = useCallback((source: number, target: number) => {
+    if (source >= 0 && target >= 0) {
+      const previousFields = cloneDeep(items.map(({item}) => item))
+      const newFields = update(previousFields, {
+        $splice: [[source, 1], [target, 0, previousFields[source]]],
+      })
+      onChange(newFields)
+    }
+  }, [items, onChange])
+
+  const droppableId = useRef(Date.now().toString())
+
+  const renderDraggable: DraggableChildrenFn = useCallback(
+    (p, s, r) => (
```

Comment:
do these params come from https://react-beautiful-dnd.netlify.app/ ? can we specify better names ?